Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM ruby:2.4.1
+ENV HOME=/usr/src/quantum
```

Comment:
/usr/src/quantum is local to the Docker container! People can have their code wherever they want :)